src/components/button-group/button-group.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { IButtonGroupProps } from "../../types";
+
+import "./button-group.scss";
+
+
+export const ButtonGroup = (props: IButtonGroupProps) => {
+  const { buttons, selectedButtonLabel, disabled, onButtonClicked } = props;
+
+  const handleClick = (i: number) => {
+    if (onButtonClicked) {
+      onButtonClicked(i, buttons[i]);
+    }
+  };
+
+  const cssClasses = (buttonLabel: string, isDisabled = false) => {
+    let classes = "buttonGroupButton";
+
+    if (buttonLabel === selectedButtonLabel) { classes += " active"; }
+    if (isDisabled) { classes += " disabled"; }
+
+    return classes;
+  };
```

Comment:
Personally, I would use the [classNames](https://www.npmjs.com/package/classnames) library for this kind of class string manipulation, but since it's not already being explicitly imported (it is being imported as a transitive dependency) it could be done at a later date.